### ISSUE TITLE
include localused SVI in only_vlans_in_use

### ISSUE THREAD

**Issue by PieterL75** [aristanetworks/ansible-avd#4521]
### Enhancement summary

SVI used for local device traffic are not accounted for in only_vlans_in_use.
ex: when an SVI is used for flow tracking, and the vlan is an EVPN vlan without being member of a connected_endpoint.
In that case the VLAN (and derived VRF) should also be added to device configuration

### Which component of AVD is impacted

eos_designs

### Use case example

I use an SVI and a dedicated VRF for my flowtracking destination. The vlan assigned to the SVI does exists on the device, but is only part of the vxlan interface, not on any connected endpoints. (except where the destination server resides ofcourse)

### Describe the solution you would like

In this case, the VLAN should also be created on the device, including the VRF related to that SVI.

### Contributing Guide

- [X] I agree to follow this project's Code of Conduct

**Comment by PieterL75** [comment]
I actually go a better solution for this.. In stead of a VLAN/SVI, I used a Loopback interface.
the VRF is added, and I don't need a VLAN anymore.. This even a more elegant solution

**Comment by ClausHolbechArista** [comment]
Agreed a loopback is better for this. Thank you for closing the loop.
